python/openchange/migration/__init__.py

```diff
@@ -118,7 +118,7 @@ def unapply_version(self, app, version):
         self.db.commit()
         self._version[app] = None  # To reload the value
 
-    def migrate(self, app, target_version=None):
+    def migrate(self, app, target_version=None, extra_params=None):
```

Comment:
I'd use `**kwargs` to let the subclass to define the named parameter with name meaningful instead of generic `extra_params`.